Samples/ResourceManagement/cpp-console-unpackaged/console_unpackaged_app.cpp

```diff
@@ -59,6 +60,13 @@ int wmain(int argc, wchar_t* argv[])
         return 1;
     }
 
+    // Despite the name of this API, it will return FALSE if the user is an admin but the context isn't elevated.
+    if (IsUserAnAdmin())
+    {
+        std::wcout << "Windows App SDK doesn't currently support elevation for unpackaged apps. See https://github.com/microsoft/WindowsAppSDK/issues/567." << std::endl;
```

Comment:
`for unpackaged apps`

Dynamic Dependencies is the issue, thus unpackaged apps using WinAppSDK-as-MSIX. If/when SelfContained unpackaged apps are supported this wouldn't be relevant (purpose in life is to give an unpackaged app access to packaged goods. No WinAppSDK/MSIX package then no need for WinAppSDK's DynDep).

SUGGESTION: "Windows app SDK DynamicDependencies doesn't support elevation. See Issue #567 https://github.com/microsoft/WindowsAppSDK/issues/567"